utils/build/docker/php/common/php.ini

```
@@ -31,3 +30,6 @@ datadog.remote_config_poll_interval=500
 datadog.remote_config_enabled=1
 datadog.experimental_api_security_enabled=1
 datadog.api_security_request_sample_rate=1
+
+datadog.trace.log_file=/tmp/tracer.log
```

With that, you'll have the logs exported to the host logs



```suggestion
datadog.trace.log_file=/var/log/system-tests/tracer.log
```